### PR TITLE
directions how to fix code style

### PR DESCRIPTION
**Pull request by danewalton**

### CODE REVIEW

**Comment by danewalton** [comment]
Example what this looks like.

image

File: .github/workflows/ci_tests_linux.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
           ( github.event.pull_request.head.repo.fork == true )
 
       - name: Style Check
-        run: .github/scripts/code_style.sh check
+        run: .github/scripts/code_style.sh check || (echo -e '::error There are files which do not adhere to the coding style. Please see directions to fix this here:" https://github.com/Azure/azure-iot-middleware-freertos#code-style' && exit 1)
```
Comment (hihigupt) [inline .github/workflows/ci_tests_linux.yml:31]: char(") does not have closing char(")

Comment (hihigupt) [reply .github/workflows/ci_tests_linux.yml:31]: can you also break this line... so that line is not more then 100 char

Comment (hihigupt) [reply .github/workflows/ci_tests_linux.yml:31]: space between ( and character

Comment (danewalton) [reply .github/workflows/ci_tests_linux.yml]: The solo `"` is an escape I added a comment detailing that.

**Review by hihigupt — APPROVED** [review]

**Pull request merged**